src/main/resources/bidder-config/bidmyadz.yaml

```diff
@@ -0,0 +1,27 @@
+adapters:
+  bidmyadz:
+    enabled: false
+    endpoint: http://endpoint.bidmyadz.com/c0f68227d14ed938c6c49f3967cbe9bc
+    pbs-enforces-gdpr: true
+    pbs-enforces-ccpa: true
+    modifying-vast-xml-allowed: true
+    deprecated-names:
+    aliases: {}
+    meta-info:
+      maintainer-email: contact@bidmyadz.com
+      app-media-types:
+        - banner
+        - video
+        - native
+      site-media-types:
+        - banner
+        - video
+        - native
+      supported-vendors:
+      vendor-id: 724
```

Comment:
This is between vendor id. BidMyAdz id is not present in go PR, and also 79.json does not contain it, so set it as zero,
Also pls fix it for bidmachine which id is 736